projects/storefrontapp-e2e-cypress/cypress/e2e/regression/asm/customer360-e2e.cy.ts

```diff
@@ -245,5 +245,19 @@ context('Assisted Service Module', () => {
           cy.get('button').contains('Apply to Cart').should('be.visible');
         });
     });
+    it('should contain promotion list (CXSPA-3932)', () => {
+      cy.get('cx-asm-customer-promotion').scrollIntoView().should('be.visible');
+      cy.get('.cx-asm-customer-promotion-listing').should('not.contain','Promotion Applied');
+    });
+    it('should be able to autoApply promotion to cart (CXSPA-3932)', () => {
+      checkout.goToProductDetailsPage();
+      Cypress._.range(0,50).forEach((index: number) =>{
```

Comment:
is it possible to type the number as 50 instead of clicking `add to cart` button 50 times?